### ISSUE TITLE
Field Width Repeater Meta Box

### ISSUE THREAD

**Issue by joseehilton147** [Crocoblock/suggestions#1393]
Let us allow to inline fields on repeater, like, i have 3 fields to repeat and want them in one line not 3, this will be more convenient and clean on a small repeater field.

![image](https://user-images.githubusercontent.com/53986027/83907842-70c42300-a73c-11ea-8f80-41b4d3002220.png)

**Comment by IrishDesigner** [comment]
I agree, it would be great to be able to put fields into a horizontal line and assign the widths of them ... all they would need to do is duplicate the field width option that they have on all the other fields
